docs/question/html.md

```diff
@@ -120,12 +144,25 @@ adding the `mathjax_ignore` class to an HTML element.
 
 ## Rendering panels from `question.html`
 
-When a question is displayed to a student, there are three "panels" that will be shown at different stages: the "question" panel, the "submission" panel, and the "answer" panel. These display the question prompt, the solution provided by the student, and the correct answer.
+When a question is displayed to a student, there are three "panels" that will be shown at different stages: the `"question"` panel, the `"submission"` panel, and the `"answer"` panel. These display the question prompt, the solution provided by the student, and the correct answer.
+
+![Question panels](./panels.png)
+
+The `"question"` panel is displayed when the question is first shown to the student. The `"submission"` panel is displayed after the student submits an answer (but before they finish attempting the question), and the `"answer"` panel is displayed when the student either submits the correct answer or runs out of attempts. You can see the [lifecycle diagram](server.md#question-lifecycle) for more details on how these panels are displayed.
```

Comment:
```suggestion
The `"question"` panel is displayed when the question is first shown to the student. A `"submission"` panel is displayed each time the student saves an answer; there may be multiple submission panels if the student submits multiple answers to the same variant. The `"answer"` panel is displayed based on question/assessment configuration; by default, it is shown when the student either submits the correct answer or runs out of attempts. You can see the [lifecycle diagram](server.md#question-lifecycle) for more details on how these panels are displayed.
```

Reply:
I applied this change in 189fcab7d634a129d31bfb5c5fd1de4e2ee3cbd9.

Reply:
In 04b49db60cb4d610934886f88dd8ca08c1714a47 I updated the lifecycle diagram to reflect this.